config.json

```diff
@@ -517,6 +523,12 @@
             "ContainerName": "default"
           }
         },
+        "nodeReuse":{
```

Comment:
What I would like to avoid is forcing every command to remember to have this alias and eliminate some of the duplication. I also want to make sure we don't accidentally start leaving open instances of msbuild so ideally I would like to see this nodereuse option right next to maxcpucount, so we know they are always paired.